spec/searchComponentSpec.js

```diff
@@ -604,13 +606,57 @@ describe('gameSearchCtrl',function(){
         !(code >= 214 && code <= 216) &&
         !(code >= 224 && code <= 229) &&
         !(code >= 233 && code <= 237) &&
+        !(code == 58) && // colon
```

Comment:
you should probably delete this an include the actual code from your angular app